app/displays.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+
+from app.models import Book
+
+
+class BookDisplay(ABC):
+    @staticmethod
+    @abstractmethod
+    def display(book: Book) -> None:
+        ...
```

Comment:
Purpose: `pass` is a null statement that acts as a placeholder in your code where a statement is syntactically required, but you don't want to perform any action. It is commonly used in function definitions, loops, and conditionals.
Purpose: `...` (Ellipsis) is also used as a placeholder but has broader uses beyond just acting as a null statement. It is more commonly seen in contexts like type hinting, slicing, and as a signal that code needs to be implemented in the future.

Looks like it's better to use pass in this case, since we don't want to implement this code in the future, but instead a statement is syntactically required, but you don't want to perform any action